api/init-test.sh

```
@@ -0,0 +1,42 @@
+if [ -z "${MYSQL_ROOT_PASSWORD}" ]; then
+    echo "ERROR: MYSQL_ROOT_PASSWORD environment variable is not set" >&2
+    exit 1
+fi
+
+if [ -z "${MYSQL_PASSWORD}" ]; then
+    echo "ERROR: MYSQL_PASSWORD environment variable is not set" >&2
+    exit 1
+fi
+
+# Execute MySQL commands with explicit error handling
+mysql -u root -p"${MYSQL_ROOT_PASSWORD}" <<-EOSQL || {
```

Shellcheck (0.10.0)

[error] Couldn't parse this brace group. Fix to allow more checks.

(SC1073)